package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@oracle/quicksql",
   "version": "1.1.2",
```

Comment:
IMO bumping up version with this change is warranted

Reply:
This are just build and documentation changes, though, so no version bump needed